eZ/Publish/API/Repository/Tests/RoleServiceTest.php

```diff
@@ -1708,7 +1745,7 @@ public function testGetRoleAssignments()
 
         /* END: Use Case */
 
-        $this->assertEquals(1, count($roleAssignments));
+        $this->assertEquals(2, count($roleAssignments));
```

Comment:
Where does this come from, AFAICS the fixture or logic for this method did not change?